docs/Manual/07 Plans - Vectorization and Parallelization.md

```diff
@@ -44,12 +44,12 @@ for j in range(5):
 for j in range(5):
     A[2, j] *= 2.0
 ```
-And, of course, we could also unroll both dimensions, removing for-loops completely.
+And, of course, we can also unroll both dimensions, removing for-loops completely.
 
 ## `vectorize`
-Many modern target platforms support SIMD vector instructions. SIMD instructions perform the same operation on an entire vector of elements, all at once. By default, each dimension of an iteration space becomes a for-loop, but the `vectorize` instruction labels a dimension for vectorized execution, rather than for-looping.
+Many modern and advanced target platforms support SIMD vector instructions. These instructions perform the same operation on an entire vector of elements, all at once. By default, each dimension of an iteration space becomes a for-loop. However, the `vectorize` instruction labels a dimension for vectorized execution, rather than for-looping.  
```

Comment:
"modern target platforms" will suffice, because SIMD is actually very common (in Intel chips), so one won't consider it an advanced platform these days. (The usage of SIMD in assembly language is advanced but that's what Accera tries to solve! :))